Tools/Pipeline/Controls/PropertyGridTable.cs

```diff
@@ -196,7 +196,7 @@ private void Drawable_Paint(object sender, PaintEventArgs e)
             if(_edit)
             {
                 if (!Global.Unix)
-                    _selectedCell.Edit(pixel1);
+                    _selectedCell?.Edit(pixel1);
```

Comment:
The null check here is pointless, _selectedCell in this part of the code cannot be null because _edit will only get set if _selectedCell is not null.

https://github.com/Linchy/MonoGame/blob/016a5abf3e0651cb9aaa9a81d44dfd807117f180/Tools/Pipeline/Controls/PropertyGridTable.cs#L237